extension/src/paymentHandler/make-payment.handler.js

```diff
@@ -13,32 +13,14 @@ async function execute (paymentObject) {
     pU.createSetCustomFieldAction(c.CTP_CUSTOM_FIELD_MAKE_PAYMENT_RESPONSE, response)
   ]
 
-  // eslint-disable-next-line default-case
-  switch (response.resultCode) {
-    case 'Authorised':
-      actions.push(
-        pU.createAddTransactionAction({
-          type: 'Authorization',
-          state: 'Success',
-          amount: paymentObject.amountPlanned.centAmount,
-          currency: paymentObject.amountPlanned.currencyCode,
-          interactionId: response.pspReference
-        })
-      )
-      break
-    case 'Refused':
-    case 'Error':
-      actions.push(
-        pU.createAddTransactionAction({
-          type: 'Authorization',
-          state: 'Failure',
-          amount: paymentObject.amountPlanned.centAmount,
-          currency: paymentObject.amountPlanned.currencyCode,
-          interactionId: response.pspReference
-        })
-      )
-      break
-  }
+  const addTransactionAction = pU.createAddTransactionActionByResponse(
+    paymentObject.amountPlanned.centAmount,
```

Comment:
why not passing amountPlanned object instead of passing centAmount, currencyCode ?

Reply:
1. There's not much difference
2. I have another method `createAddTransactionAction` where I also pass centAmount and currencyCode separately, so for consistency.

Reply:
Anyway, it does not add much value, in the end, it's the same. Please pass that comment.